tests/test_pipeline/test_stage_manager.py

```diff
@@ -78,9 +78,9 @@ def run_dist(rank, world_size, port):
 
 
 @pytest.mark.dist
-def test_process_group_mesh():
+def test_pipeline_stage_manager():
```

Comment:
add the `rerun_if_address_in_use` decorator (if I didn't spell wrongly)?

Reply:
done